cmake/Dependencies.cmake

```diff
@@ -1324,6 +1324,52 @@ if(USE_ROCM AND ROCM_VERSION_DEV VERSION_LESS "5.2.0")
   include_directories(SYSTEM ${THRUST_PATH})
 endif()
 
+# ---[ XPU
+if(USE_XPU)
+  include(${CMAKE_CURRENT_LIST_DIR}/public/xpu.cmake)
+  if(PYTORCH_FOUND_XPU)
+    # # -- Host flags (SYCL_CXX_FLAGS)
+    # Combine with torch_compile_options
+    list(APPEND SYCL_HOST_FLAGS -fPIC)
+    list(APPEND SYCL_HOST_FLAGS -std=c++17)
+    # SYCL headers warnings
+    list(APPEND SYCL_HOST_FLAGS -Wno-deprecated-declarations)
+    list(APPEND SYCL_HOST_FLAGS -Wno-attributes)
+
+    if(CMAKE_BUILD_TYPE MATCHES Debug)
+       list(APPEND SYCL_HOST_FLAGS -g2)
```

Comment:
Although `-g2` and `-g` are equivalent, please follow PyTorch by replacing `-g2` with `-g`.